templates/docs/examples/patterns/image/spacing.html

```diff
@@ -0,0 +1,13 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Image / Spacing{% endblock %}
+
+{% block standalone_css %}base{% endblock %}
+
+{% block content %}
+<figure>
```

Comment:
I'm not sure if it's valid for `figure` to contain elements that are not an image and caption.

I think we can just drop it from this example, and keep it minimal (so just the `hr` rule and image in the container.